doc/source/CodeFormatting.rst

```diff
@@ -0,0 +1,106 @@
+Style Formatting
+================
+
+The Grackle repository is configured with tools to enforce checks (such as running the linter or applying code-formatting) on the various files in the repository.
+
+The primary checks pertain to python code: all python code is linted and all new python files are formatted by automated tools (for the time being, code in older files will not be formatted to avoid merge-conflicts).
```

Comment:
```suggestion
The primary checks pertain to python code: all python code is linted and all new python files are formatted by automated tools (for the time being, code in older files will not be formatted to avoid merge conflicts).
```